admin_manual/configuration_server/reverse_proxy_configuration.rst

```diff
@@ -136,7 +136,20 @@ Caddy
 
         reverse_proxy {$NEXTCLOUD_HOST:localhost}
     }
-
+    
+Pomerium
+^^^^^^^^
+::
+    - from: https://subdomain.example.com
+      path: /.well-known/carddav
+      redirect:
+        path_redirect: /remote.php/dav/
+    - from: https://subdomain.example.com
+      path: /.well-known/caldav
+      redirect:
+        path_redirect: /remote.php/dav/
+
+Thanks to `@JeffMatson <https://github.com/JeffMatson>`_ for Pomerium example.
```

Comment:
you can add a https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-sectionauthor but not in a visible way. The documentation has been written by lots of people https://github.com/nextcloud/documentation/graphs/contributors :)

Reply:
I fear I introduce the "Thanks to" pattern with https://github.com/nextcloud/documentation/pull/1297 :see_no_evil: 

The rules were from a GitHub issue, and my motivation was to acknowledge the actual authors.